pages/iam/how-to/comply-with-sec-requirements-member.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: How to comply with security requirements as a Member
+description: Instructions for complying with security requirements as an IAM Member
+content:
+  h1: How to comply with security requirements as a Member
+  paragraph: Instructions for complying with security requirements as an IAM Member
+dates:
+  validation: 2025-03-24
+  posted: 2025-03-24
+---
+
+Upon your [first login as a Member](/iam/how-to/log-in-as-a-member), you must comply with your Organization's security requirements to ensure you can log in without issues in the future.
+
+<Macro id="requirements" />
+
+- A Scaleway account and logged into the [console](https://console.scaleway.com) as an [IAM Member](/iam/concepts/#member)
+
+## How to check the security requirements
+
+When you log in for the first time, a security checklist displays in your Scaleway console.
+
+<Lightbox src="scaleway-iam-member-sec-req.webp" alt="screenshot of the scaleway console showing a checklist of the few quick steps to follow until you can explore the Scaleway console. 1. Update password - You are required to update your password to stay connected to this Organization. A message box indicated that there is 1 day left to update the password. It warns that once this grace period is up, you will be locked out of the Organization until an administrator unlocks your account. A button prompts to update the password. "/>
+
+Currently, the mandatory security requirements include:
+
+- [Updating your password](#how-to-update-a-password)
+- [Setting up MFA](#how-to-set-up-mfa-as-a-member)
+
+### Grace period
+
+New IAM Members have a [grace period](/iam/concepts/#grace-period) available to comply with security requirements. The grace period is defined by the Organization's administrator or is set to default (7 days).
+
+<Message type="important">
+Once the grace period is over, your Member account is automatically locked and you must personally contact the Organization administrator to unlock you. When you regain access to the Organization, the grace period resets and you have the set amount of time to comply with the requirements again.
+</Message>
+
+For example, if your Organization's grace period is set to default, you have 7 days, starting from your first login, to renew your password or define a new one, and to set up MFA. If you fail to comply until the 11:59 p.m. of the 7th day, you will get locked out of the Organization at 00:00 a.m. of the 8th day.
```

Comment:
@aerialls I think it's 7 days starting at the time of the request. Like if I am created at 3pm on a Monday, I would be locked next Monday at 3pm, can you confirm it Julien?

Reply:
Yes I confirm